joomla/joomla-cms
# [4.3] Add a Global Checking Task plugin

### Summary of Changes

Add a Global Checking Task plugin as an adaption of my recently published plg_task_checkin plugin (https://github.com/zero-24/plg_task_checkin).
As agreed with @obuisard this here is a PR against 4.3 to be shipped with 4.3.0

### Testing Instructions

- Install the patch
- run the extension discover
- publish the plg_task_globalcheckin plugin
- create a new task
- notice there is a new option called "Global Checkin"
- set it up to be manually triggered
- checkout some articles
- verify that something is checked out within com_checkin
- manually trigger the global checkin task
- notice the message how many items are checked in
- verify that everything is checked in within com_checkin.

### Actual result BEFORE applying this Pull Request

An global checking plugin does not exist.

### Expected result AFTER applying this Pull Request

An global checking task plugin is part of the core to support site maintainers

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed as no page exists yet

### Thanks

@brianteeman and @Worti2
